src/screens/MyListsScreen.tsx

```diff
@@ -46,11 +47,36 @@ class MyListsScreen extends React.Component<Props, State> {
     });
   }
 
-  componentDidMount() {
+  async componentDidMount() {
     this.refresh();
     this._unsubscribe = this.props.navigation.addListener('focus', () => {
       this.fetchLists();
     });
+    Audio.setAudioModeAsync({
+      allowsRecordingIOS: false,
+      interruptionModeIOS: Audio.INTERRUPTION_MODE_IOS_DO_NOT_MIX,
+      playsInSilentModeIOS: true,
+      interruptionModeAndroid: Audio.INTERRUPTION_MODE_ANDROID_DUCK_OTHERS,
+      shouldDuckAndroid: true,
+      staysActiveInBackground: true,
+      playThroughEarpieceAndroid: true,
```

Comment:
```suggestion
      playThroughEarpieceAndroid: false,
```
This causes the sound to be played through the top call speaker instead of the main speaker.